apifest-api/src/main/java/com/apifest/api/MappingDocumentation.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2013-2014, ApiFest project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apifest.api;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "mappings_docs")
+public class MappingDocumentation implements Serializable {
+
+    private static final long serialVersionUID = 8027603159873624488L;
+
+    @XmlAttribute(name = "version", required = true)
+    private String version;
+
+    @XmlElement(name = "endpoints", type = MappingEndpointDocumentation.class, required = true)
+    private List<MappingEndpointDocumentation> mappingEndpontDocumentation;
```

Comment:
Please, fix the typo in the name of the variable - one "i" is missing in the endpoint part of the name.